internal/api/api.go

```diff
@@ -24,9 +25,9 @@ type CollageRequest struct {
 	Period        string `in:"query=period;default=7day" validate:"required,validatePeriod"`
 	Height        uint   `in:"query=height;default=0" validate:"gte=0,lte=3000"`
 	Width         uint   `in:"query=width;default=0" validate:"gte=0,lte=3000"`
-	Rows          int    `in:"query=rows;default=3" validate:"required,gte=1,lte=20"`
+	Rows          int    `in:"query=rows;default=3" validate:"required"`
```

Comment:
Also I noticed you removed `lte` and `gte`—is the idea to set the max by the amount of images and not grid size?